src/coreclr/jit/hwintrinsiclistarm64.h

```diff
@@ -316,6 +316,9 @@ HARDWARE_INTRINSIC(AdvSimd,       InsertScalar,
 HARDWARE_INTRINSIC(AdvSimd,       LeadingSignCount,                                                 -1,      1,      true,  {INS_cls,            INS_invalid,        INS_cls,            INS_invalid,        INS_cls,            INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_NoFlag)
 HARDWARE_INTRINSIC(AdvSimd,       LeadingZeroCount,                                                 -1,      1,      true,  {INS_clz,            INS_clz,            INS_clz,            INS_clz,            INS_clz,            INS_clz,            INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_NoFlag)
 HARDWARE_INTRINSIC(AdvSimd,       LoadAndInsertScalar,                                              -1,      3,      true,  {INS_ld1,            INS_ld1,            INS_ld1,            INS_ld1,            INS_ld1,            INS_ld1,            INS_ld1,            INS_ld1,            INS_ld1,            INS_ld1},         HW_Category_MemoryLoad,            HW_Flag_HasImmediateOperand|HW_Flag_HasRMWSemantics|HW_Flag_SIMDScalar|HW_Flag_SpecialCodeGen)
+HARDWARE_INTRINSIC(AdvSimd,       LoadAndInsertScalarx2,                                             8,      3,      true,  {INS_ld2,            INS_ld2,            INS_ld2,            INS_ld2,            INS_ld2,            INS_ld2,            INS_invalid,        INS_invalid,        INS_ld2,            INS_invalid},     HW_Category_MemoryLoad,            HW_Flag_HasImmediateOperand|HW_Flag_SpecialImport|HW_Flag_HasRMWSemantics|HW_Flag_SIMDScalar|HW_Flag_SpecialCodeGen|HW_Flag_MultiReg|HW_Flag_NeedsConsecutiveRegisters)
```

Comment:
It seems like we should name these `LoadAndInsertScalarVector64xN` and the `AdvSimd_Arm64` versions `LoadAndInsertScalarVector128xN` to be absolutely clear the size of the individual vector elements (so you don't need to "just know" the vector element size from the namespace.